src/Component.php

```diff
@@ -4,34 +4,55 @@
 
 namespace Keboola\SiSenseWriter;
 
-use GuzzleHttp\Client;
 use Keboola\Component\BaseComponent;
-use Keboola\SiSenseWriter\Api\Api;
+use Keboola\Component\UserException;
+use Keboola\SiSenseWriter\Configuration\TestConnectionConfigDefinition;
+use Keboola\SiSenseWriter\Configuration\ConfigDefinition;
 
 class Component extends BaseComponent
 {
+    private const ACTION_RUN = 'run';
+
+    private const ACTION_TEST_CONNECTION = 'testConnection';
+
     protected function run(): void
     {
         /** @var Config $config */
         $config = $this->getConfig();
+        $componentFactory = new ComponentFactory($this->getLogger());
```

Comment:
V `Component` volas `ComponentFactory` ... dost je to matuce ... kedze `ComponentFactory` by mala vytvarat `Component` ... teda asi by sa to malo volat `WriterFactory` alebo `SiSenseWriterFactory` ....

Reply:
🤦 máš pravdu :)